Library/Formula/fq.rb

```diff
@@ -0,0 +1,21 @@
+class Fq < Formula
+  desc "A brokered message queue written in C"
+  homepage "https://github.com/circonus-labs/fq"
+  url "https://github.com/circonus-labs/fq/archive/v0.9.1.tar.gz"
+  sha256 "9fd8d51211a5ebe776c0e4e61a9d4475ba39e4597fe5cd45ea9a1043da3cf4ef"
+
+  depends_on "concurrencykit"
+  depends_on "jlog"
+  depends_on "openssl"
+
+  def install
+    inreplace "Makefile", "/usr/lib/dtrace", "#{lib}/dtrace"
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+    bin.install "fqc", "fq_sndr", "fq_rcvr"
+  end
+
+  test do
+    system "fqd", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!